gobblin-modules/gobblin-elasticsearch/scripts/install_test_deps.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+TARGET_DIR="test-elasticsearch"
+ES_VERSION=5.6.8
```

Comment:
should this be passed along from gradle script as param, where gralde script 'knows' about it through project properties (as versions for deps are defined in gradle scripts)

Reply:
Not sure if it can be made that generic and configurable... the download locations for different versions of elasticsearch were different when I spot-checked a couple.

Reply:
Not going to do this... too much work.

Reply:
nw, create a Jira :)